src/main/java/com/synopsys/integration/detect/lifecycle/boot/DetectBootFactory.java

```diff
@@ -124,7 +124,8 @@ public DiagnosticSystem createDiagnosticSystem(boolean isDiagnosticExtended, Pro
         return new DiagnosticSystem(isDiagnosticExtended, detectConfiguration, detectRunId, detectInfo, directoryManager, eventSystem, maskedRawPropertyValues, propertyKeys);
     }
 
-    public AirGapCreator createAirGapCreator(ConnectionDetails connectionDetails, DetectExecutableOptions detectExecutableOptions, Configuration freemarkerConfiguration) {
+    public AirGapCreator createAirGapCreator(ConnectionDetails connectionDetails, DetectExecutableOptions detectExecutableOptions, Configuration freemarkerConfiguration, InstalledToolManager installedToolManager,
+        InstalledToolLocator installedToolLocator) {
```

Comment:
When method parameters wrap to multiple lines, we prefer to move the arguments to separate lines. IntelliJ has a shortcut for this.

Reply:
like a separate line for each?  do you know the shortcut off the top of your head?